src/SignUpIn/DonatingAgency/DonatingAgencySignUpController.js

```diff
@@ -60,6 +62,62 @@ class DonatingAgencySignUpController extends Component {
         });
     }
 
+    submitRegistration() {
+        // Handle via ajax submitting the user data, upon
+        // success return this.nextStop(). If it fails,
+        // show the user the error but don't advance
+
+        firebase.auth().createUserWithEmailAndPassword(fieldValues.adminEmail, fieldValues.adminPassword)
+            .then(user => {    
+                let agencyKey = firebase.database().ref().child('accounts').push().key;
```

Comment:
agency should be pushed to `'donating_agencies'`

Reply:
Fixed

Reply:
Fixed.